draft-ietf-quic-recovery.md

```diff
@@ -1179,20 +1179,20 @@ OnLossDetectionTimeout():
     SetLossDetectionTimer()
     return
 
-  if (endpoint is client without 1-RTT keys):
+  if (bytes_in_flight > 0):
+    // PTO. Send new data if available, else retransmit old data.
+    // If neither is available, send a single PING frame.
+    _, pn_space = GetEarliestTimeAndSpace(
+      time_of_last_sent_ack_eliciting_packet)
+    SendOneOrTwoAckElicitingPackets(pn_space)
+  else:
```

Comment:
```suggestion
  else if client:
```

Otherwise, the server might hit this path if it has sent HelloRetryRequest and had it acknowledged.  In that case, the server has to sit tight.  It's a perverse scenario, but PQ is coming...

Reply:
Oh, this might need all the original constraints as well, otherwise this path will be hit by a quiescent client all the time.

Reply:
The alarm shouldn't be armed in those case, as stated above in the normative text
"If no data can be sent, then the PTO alarm MUST NOT be armed until datagrams have been received from the client."

Would you like some pseudocode added to the "Setting the Loss Detection Timer" pseudocode?